packages/app/src/components/search/GithubDiscussionsSearchResultListItem.tsx

```diff
@@ -0,0 +1,128 @@
+import { HighlightedSearchResultText } from '@backstage/plugin-search-react';
+import { IndexableDocument } from '@backstage/plugin-search-common';
+import { EntityRefLink } from '@backstage/plugin-catalog-react';
+
+export interface GithubDiscussionsDocument extends IndexableDocument {
+  author: string;
+  category: string;
+  labels: {
+    name: string;
+    color: string;
+  }[];
+  comments: {
+    author: string;
+    bodyText: string;
+    replies: {
+      author: string;
+      bodyText: string;
+    }[];
+  }[];
+}
+
+const useStyles = makeStyles(theme => ({
+  item: {
+    display: 'flex',
+  },
+  flexContainer: {
+    flexWrap: 'wrap',
+  },
+  itemText: {
+    width: '100%',
+    wordBreak: 'break-all',
+    marginBottom: '1rem',
+  },
+  user: {
+    display: 'inline-flex',
+    margin: theme.spacing(1),
+  },
+}));
+
+/**
+ * Props for {@link GithubDiscussionsSearchResultListItem}.
+ *
+ * @public
+ */
+export interface GithubDiscussionsSearchResultListItemProps {
+  icon?: ReactNode;
+  result?: GithubDiscussionsDocument;
+  highlight?: ResultHighlight;
+  lineClamp?: number;
+}
+
+/** @public */
+export function GithubDiscussionsSearchResultListItem(
+  props: GithubDiscussionsSearchResultListItemProps,
+) {
+  const { result, highlight, icon } = props;
+  const classes = useStyles();
+  if (!result) return null;
+
+  return (
+    <div className={classes.item}>
+      {icon && <ListItemIcon>{icon}</ListItemIcon>}
+      <div className={classes.flexContainer}>
+        <ListItemText
+          className={classes.itemText}
+          primaryTypographyProps={{ variant: 'h6' }}
+          primary={
+            <Link noTrack to={result.location}>
+              {highlight?.fields.title ? (
+                <HighlightedSearchResultText
+                  text={highlight.fields.title}
+                  preTag={highlight.preTag}
+                  postTag={highlight.postTag}
+                />
+              ) : (
+                result.title
+              )}
+            </Link>
+          }
+          secondary={
+            <Typography
+              component="span"
+              style={{
+                display: '-webkit-box',
+                WebkitBoxOrient: 'vertical',
+                WebkitLineClamp: props.lineClamp,
+                overflow: 'hidden',
+              }}
+              color="textSecondary"
+              variant="body2"
+            >
+              {highlight?.fields.text ? (
+                <HighlightedSearchResultText
+                  text={highlight.fields.text}
+                  preTag={highlight.preTag}
+                  postTag={highlight.postTag}
+                />
+              ) : (
+                result.text
+              )}
+            </Typography>
+          }
+        />
+        <Box>
+          {result.author && (
+            <div className={classes.user}>
+              <EntityRefLink entityRef={`user:default/${result.author}`} />
```

Comment:
Is there any scenario where `user:default` would not be the namespace? Is it okay to hard code this here?

Reply:
@minkimcello - it's all good, I'll make as resolved.